setup/install_upstream_rust_keylime/test.sh

```diff
@@ -25,11 +26,26 @@ rlJournalStart
                 [ -f ${FILE} ] && rlRun "sed -i 's%/sys/kernel/security/tpm0/binary_bios_measurements%${TPM_BINARY_MEASUREMENTS}%' $FILE"
             done
         fi
+
+        if [ -n "$KEYLIME_RUST_CODE_COVERAGE" ]; then
```

Comment:
I think better to use
```
[ "$KEYLIME_RUST_CODE_COVERAGE" == "1" -o "$KEYLIME_RUST_CODE_COVERAGE" == "true" ]
```
so we can be possibly explicit in a test plan stating `KEYLIME_RUST_CODE_COVERAGE: false` or similar.